src/NServiceBus.Persistence.AzureStorage/SagaPersisters/SagaIdGenerator.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.Persistence.AzureStorage
+{
+    using System;
+    using Sagas;
+    using System.Security.Cryptography;
+    using System.Text;
+    using Newtonsoft.Json;
+
+    class SagaIdGenerator : ISagaIdGenerator
```

Comment:
Do we need to make it public?

Reply:
I'd say we wait. we can do that if necessary in another PR